Tests/TestingTests/MiscellaneousTests.swift

```diff
@@ -389,19 +389,6 @@ struct MiscellaneousTests {
     #expect(tests.map(\.displayName) == ["A", "B", "C", "D", "E", "F", "G"])
   }
 
-  @Test("Parameterizing using lazy collections",
-    .disabled("Lazy collections are not sendable, so are not selected during overload resolution (favoring arrays instead.)"),
-    .bug("rdar://104169208")
-  )
-  func testParameterizedOverLazyCollection() async throws {
-    // Test that functional transformations on the collection are supported.
-    let range = 1 ... 100
-    let test = Test(arguments: range.lazy.map(-)) { i in
```

Comment:
Do you have any more context/background about what changed with this one?

Reply:
The warning is just louder AFAICT. This was always an unsafe test, which is why it was disabled to begin with.